airbyte-integrations/bases/base-normalization/Dockerfile

```diff
@@ -1,4 +1,20 @@
 FROM fishtownanalytics/dbt:0.19.0
+
+USER root
+WORKDIR /tmp
+RUN apt-get update && apt-get install -y \
+    wget \
+    unzip \
+    libaio-dev \
+    libaio1
+RUN mkdir -p /opt/oracle
+RUN wget https://download.oracle.com/otn_software/linux/instantclient/19600/instantclient-basic-linux.x64-19.6.0.0.0dbru.zip
+RUN unzip instantclient-basic-linux.x64-19.6.0.0.0dbru.zip -d /opt/oracle
+ENV ORACLE_HOME /opt/oracle/instantclient_19_6
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$ORACLE_HOME
+ENV TNS_ADMIN /opt/oracle/instantclient_19_6/network/admin
```

Comment:
does this work if we run integration tests with oracle v12? or should we update the docs to be only v19+?

Reply:
Not supported it's possible to increase the limit but when creating the database is not available, I'd update docs to v18+